src/utils.js

```diff
@@ -36,3 +64,165 @@ export const courseDateFormat = (course) => {
     };
 };
 
+export const sessionPaymentStatus = (session, enrollment) => {
+    const session_date = new Date(session.start_datetime),
+        last_session = new Date(enrollment.last_paid_session_datetime);
+
+    const sessionIsBeforeLastPaidSession = session_date <= last_session;
+    const sessionIsLastPaidSession = session_date == last_session;
+    const thereIsPartiallyPaidSession = !Number.isInteger(enrollment.sessions_left);
+    const classSessionNotBeforeFirstPayment = session_date >= new Date(enrollment.payment_list[0].created_at);
+
+    if( sessionIsBeforeLastPaidSession && !thereIsPartiallyPaidSession && classSessionNotBeforeFirstPayment){
+        return "Paid";
+    } else if ( sessionIsLastPaidSession && thereIsPartiallyPaidSession && thereIsPartiallyPaidSession){
+        return "Partial";
+    } else if (!classSessionNotBeforeFirstPayment ) {
+        return "NA"
+    } else {
+        return "Unpaid";
+    }
+};
+
+const courseToRegister = (enrollment, course, student) => ({
+    "Enrollment": enrollment.enrollment_id,
+    "Course Selection": {
+        "Course": {
+            label: course.title,
+            value: Number(course.course_id),
+        },
+    },
+    "Course Selection_validated": {
+        "Course": true,
+    },
+    "Student": {
+        "Student": {
+            label: student.name,
+            value: student.user_id,
+        }
+    },
+    "Student_validated": {
+        "Student": true,
+    },
+    "Student Information": {},
+    "activeSection": "Student",
+    "activeStep": 0,
+    "conditional": "",
+    "existingUser": false,
+    "form": course.course_type,
+    "hasLoaded": true,
+    "preLoaded": false,
+    "submitPending": false,
+});
+
+/**
+ * @description button/menu item to start registering for more sessions for a course.
+ * This will display a warning popover in the case that you're about to override
+ * an existing registering parent.
+ * @param {String} componentOption this will either be a button or a menu item
+ * @param {Number} parentOfCurrentStudent this is the id of the parent that will be registering
+ * @param {Object} enrollment this is the enrollment we'll be updating
+ */
+export const AddSessions = ({componentOption, parentOfCurrentStudent, enrollment}) => {
+    const history = useHistory();
+    const dispatch = useDispatch();
+    const api = useMemo(
+        () => ({
+            ...bindActionCreators(registrationActions, dispatch),
+        }),
```

Comment:
Can be simply written as:

    () => bindActionCreators(registrationActions, dispatch)

since there is only one set of actions